Pages/login.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Component Library -->
+    <link
+      rel="stylesheet"
+      href="https://stackui.netlify.app/css/documentation.css"
+    />
+    <link rel="stylesheet" href="../css/login.css" />
+    <title>MenStack</title>
+  </head>
+  <body>
+    <!-- Navbar starts  -->
+    <nav id="navbar">
+      <div class="navbar-left">
+        <a href="../index.html"><h3 class="h3"><span class="primary-color">Men</span>Stack</h3></a>
+      </div>
+
+      <div class="navbar-right">
+        <a href="../Pages/signup.html"
+          ><button class="btn-link btn btn-primary">Register</button></a
+        >
+      </div>
+    </nav>
+
+    <!-- Hero starts -->
+    <section id="hero">
+      <span class="overlay"></span>
+      <form class="input-wrapper">
+        <div class="form-title">
+          <h5 class="h5">LOGIN</h5>
+        </div>
```

Comment:
suggestion - instead of writing div for h5, you can just extend the class name.

Reply:
Done as suggested.

Reply:
Thank you for pointing this out. 😃